meilisearch-http/src/main.rs

```diff
@@ -19,15 +19,25 @@ mod analytics;
 static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
 
 pub fn main() -> Result<(), MainError> {
-    env_logger::init();
 
-    let opt = Opt::from_args();
-    let data = Data::new(opt.clone());
+    let mut opt = Opt::from_args();
+
+    if opt.env == "production" {
```

Comment:
Please replace this by a match and use the [structopt possible values feature](https://github.com/TeXitoi/structopt/blob/master/examples/enum_in_args.rs) (using a const slice of str).